lib/boom_notifier.ex

```diff
@@ -1,16 +1,57 @@
 defmodule BoomNotifier do
   @moduledoc false
+
   # Notify the exception to all the defined notifiers
 
   alias BoomNotifier.ErrorStorage
 
+  def walkthrough_notifiers(settings, callback) do
+    case Keyword.get(settings, :notifiers) do
+      nil ->
+        with {:ok, notifier} <- Keyword.fetch(settings, :notifier),
+             {:ok, options} <- Keyword.fetch(settings, :options) do
+          callback.(notifier, options)
```

Comment:
Question.. here we are simply skipping if we have, for instance, only one of these `notifier` and `options` keys and the other missing. There is no message or anything to the user.

Are we leaving this kind of checking for a separate PR? Just confirming.

Reply:
Good catch! I think we should address that in this PR if @romiparada is also OK with it